docs/mobile-apps/automated-testing/appium/virtual-devices.md

```diff
@@ -20,12 +20,11 @@ With Sauce Labs, you can run automated Appium tests for your native and hybrid m
 
 ## Upload Your App to Emulators and Simulators
 
-1. To begin, you'll need to upload your app to a publicly available source. Accepted file types are *.zip iOS package files for simulators and *.apk Android package files for emulators. There are two ways you can upload your app for automated testing:
+1. To begin, upload your app to a publicly available source. Accepted file types are \*.zip iOS package files for simulators and \*.apk Android package files for emulators. There are two ways you can upload your app for automated testing:
```

Comment:
Shouldn't this contains *.ipa too ?

Reply:
I thought the same, @FriggaHel, but since that's what was already there, I wasn't sure whether it was a simulator limitation. @linafarid -- can you please confirm?

Reply:
Confirmed with the mobile team that this is correct -- .ipa files are for real devices only.